### DISCUSSION TITLE
Party sheet events

### DISCUSSION THREAD

**Discussion by m-denadai** [Lurkars/gloomhavensecretariat#244] [Q&A]
I am currently using GHS for my FH campaign and liking it very much.
I am also using the party sheet to keep track of party stats and resources.
Still, some events require us to write some section numbers in the calendar "X weeks from now". Is there any way this could be added within the app? I seem not to be able to find how to do it, if it is possible at all.

**Comment by Lurkars** [comment]
Sorry, I need to write more documentation:
Just double click (long press on mobile) on a calendar week to open menu for adding/removing sections.

<small>I am also currently working on this section rewards, like when a calendar section unlocks a new scenario for example. But don't know when this is ready.</small>

**Reply by korvegan** [reply]
Maybe add a little green plus next by mouse over in the box to indicate that you can add something. Might be worth to consider for all similar boxes
